rust/src/tests/ftruncate.rs

```diff
@@ -18,10 +18,10 @@ use crate::{
 // tests/ftruncate/00.t
 
 crate::test_case! {
-    /// ftruncate extends if the file is empty
-    increase_empty
+    /// ftruncate should shrink a sparse file
```

Comment:
This description got worse now.  The test does cover extension too.  It both extends a file, and shrinks a sparse file.